### DISCUSSION TITLE
How to consume a web API?

### DISCUSSION THREAD

**Discussion by Jerohan** [unoplatform/uno#9336] [Q&A] [answered]
Hii,

I have followed the tutorial:
https://platform.uno/docs/articles/howto-consume-webservices.html
Step by Step

But i haven't  able do it, specifically it does not detect
`using Uno.Extensions.Specialized;`

I don't know which package to install, the tutorial doesn't say anything about it
I don't know what else to do

If anyone knows another example of how consume an API.

**Comment by Paolo55555** [comment, marked as answer]
The tutorial you read is not up to date.
See the updated code in GitHub: TheCatApiClient, in particular the [WebServices folder](https://github.com/unoplatform/Uno.Samples/tree/master/UI/TheCatApiClient/TheCatApiClient/TheCatApiClient.Shared/WebServices).

**Reply by Jerohan** [reply]
thank you very much!!
